content/blog/Installation.md

```diff
@@ -1,6 +1,113 @@
+    integrity: >-
+      sha512-l1MJIrZeXp9nOQpxFF5cw1ItOgA/p4xhGjKN12sg4Re8GC1qL+5hik+lA1BjMxAN6nKGWsLdFkgqLWa6jQuQFw==
+    pluginConfig:
+      orchestrator:
+        dataIndexService:
+          url: http://sonataflow-platform-data-index-service.sonataflow-infra
+        editor:
+          path: "https://sandbox.kie.org/swf-chrome-extension/0.32.0"
+  - disabled: false
+    package: "@janus-idp/backstage-plugin-orchestrator@1.7.8"
+    integrity: >-
+      sha512-wJtu4Vhx3qjEiTe/i0Js2Jc0nz8B3ZIImJdul02KcyKmXNSKm3/rEiWo6AKaXUk/giRYscZQ1jTqlw/nz7xqeQ==
+    pluginConfig:
+      dynamicPlugins:
+        frontend:
+          janus-idp.backstage-plugin-orchestrator:
+            appIcons:
+              - importName: OrchestratorIcon
+                module: OrchestratorPlugin
+                name: orchestratorIcon
+            dynamicRoutes:
+              - importName: OrchestratorPage
+                menuItem:
+                  icon: orchestratorIcon
+                  text: Orchestrator
+                module: OrchestratorPlugin
+                path: /orchestrator
+```
+The plugin versions may change and so their integrity values. To pick the latest, refer to the [helm chart](https://github.com/parodos-dev/orchestrator-helm-chart/blob/main/charts/orchestrator/templates/rhdh-operator.yaml#L159). Make sure to set both the version and the integrity values.
+
+The `dataIndexService.url` should point to the service of the Data Index installed by the Chart/Operator.
+
+In the `app-config` ConfigMap add the following:
+```
+app:
+  backend:
+    csp:
+        frame-src:
+          - "https://sandbox.kie.org"
+      cors:
+        origin: {{ the URL for backstage}}
+```
+
+To enable the Notifications plugin, edit the same ConfigMaps.
+For the `dynamic-plugins` ConfigMap add:
+```
+      - disabled: false
+        package: "@janus-idp/plugin-notifications@1.1.12"
+        integrity: >-
+          sha512-GCdEuHRQek3ay428C8C4wWgxjNpNwCXgIdFbUUFGCLLkBFSaOEw+XaBvWaBGtQ5BLgE3jQEUxa+422uzSYC5oQ==
+        pluginConfig:
+          dynamicPlugins:
+            frontend:
+              janus-idp.backstage-plugin-notifications:
+                appIcons:
+                  - name: notificationsIcon
+                    module: NotificationsPlugin
+                    importName: NotificationsActiveIcon
+                dynamicRoutes:
+                  - path: /notifications
+                    importName: NotificationsPage
+                    module: NotificationsPlugin
+                    menuItem:
+                      icon: notificationsIcon
+                      text: Notifications
+                    config:
+                      pollingIntervalMs: 5000
+      - disabled: false
+        package: "@janus-idp/plugin-notifications-backend-dynamic@1.3.6"
+        integrity: >-
+          sha512-Qd8pniy1yRx+x7LnwjzQ6k9zP+C1yex24MaCcx7dGDPT/XbTokwoSZr4baSSn8jUA6P45NUUevu1d629mG4JGQ==
```

Comment:
what about remember to edit the version/integrity even in this case?
I'm afraid that people blindly copy&paste it otherwise.

Reply:
I'd prefer to have a specific version that works. We can add a dedicated paragraph for the upgrade.